torax/_src/geometry/tests/imas_test.py

```diff
@@ -97,6 +98,61 @@ def test_IMAS_input_with_equilibrium_object(self):
     )
     config.build_geometry()
 
+  def test_IMAS_loading_specific_slice(self):
+    filename = 'ITERhybrid_rampup_11_time_slices_COCOS17_IDS_ddv4.nc'
+    SLICE_TIME = 40
+    SLICE_INDEX = 5
+    config_at_0 = geometry_pydantic_model.IMASConfig(imas_filepath=filename)
+    config_at_slice_from_time = geometry_pydantic_model.IMASConfig(
+        imas_filepath=filename, slice_time=SLICE_TIME
+    )
+    config_at_slice_from_index = geometry_pydantic_model.IMASConfig(
+        imas_filepath=filename, slice_index=SLICE_INDEX
+    )
+    geo_at_0 = config_at_0.build_geometry()
+    geo_at_slice_from_time = config_at_slice_from_time.build_geometry()
+    geo_at_slice_from_index = config_at_slice_from_index.build_geometry()
+
+    for key in geo_at_0.__dict__.keys():
+      if key not in ['geometry_type', 'torax_mesh', 'R_major', 'B_0', 'rho_hires_norm', 'Phi_b_dot', 'Ip_from_parameters']:
+        np.testing.assert_allclose(
+            geo_at_slice_from_time.__dict__[key],
+            geo_at_slice_from_index.__dict__[key],
+            err_msg=f'Value {key} mismatched between slice_time and slice_index',
+        )
+        with self.assertRaises(AssertionError):
+          np.testing.assert_allclose(
+              geo_at_0.__dict__[key],
+              geo_at_slice_from_time.__dict__[key],
+          )
+        with self.assertRaises(AssertionError):
```

Comment:
if you've checked the two sliced geo are equal then it's not necessary to check they both don't agree with geo at t=0